Sources/Swift/Components/Dates and Timers/IdleTimer/IdleTimer+InternalTimer.swift

```diff
@@ -59,6 +64,12 @@ extension IdleTimer {
             })
         }
 
+        private func handleVOAnnouncement() {
```

Comment:
Posting the warning announcement should not be part of this class. This is just a generic timer which just calls an `onTimeout` block.